toqito/nonlocal_games/xor_game.py

```diff
@@ -70,8 +70,8 @@ class XORGame:
     >>> import numpy as np
     >>> from toqito.nonlocal_games.xor_game import XORGame
     >>> chsh = XORGame(prob_mat, pred_mat)
-    >>> chsh.quantum_value()
-    0.8535533885683664
+    >>> '%.2f' % chsh.quantum_value()
+    '0.85'
```

Comment:
How to clarify why we use `'%.2f' % ` in the output?

This is a workaround to doctest's float failures. With the new changes, docstring expects a string `'0.85'` instead of a float ` 0.8535533885683664`.

Reply:
If you're asking for clarification as to why we truncate, part of the reason for that is the accuracy of the solvers, as they are only able to sometimes get a few digits of accuracy before deviating. Does that answer the question?

Reply:
Kinda. I think we might have to have a one liner statement for whenever we use something like `'%.2f' %` in the doc examples. 

I am making `doctest` compare a string in the screenshots but a user does not have to do so. I don't know how to clarify this in every example.